<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-47</y>
        <width>780</width>
        <height>403</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Icons theme:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="iconsWButton">
              <property name="text">
               <string>White</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="iconstBButton">
              <property name="text">
               <string>Black</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Use gradient in portrait mode:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="gradientYButton">
              <property name="text">
               <string>Yes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="gradientNButton">
              <property name="text">
               <string>No</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Active track color:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="cButtonsLayout" rowstretch="0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="cBlueButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(100, 150, 220)</string>
              </property>
              <property name="text">
               <string>blue</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="cMagentaButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(150, 80, 150);</string>
              </property>
              <property name="text">
               <string>magenta</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="cRedButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(220, 100, 100);</string>
              </property>
              <property name="text">
               <string>red</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QRadioButton" name="cYellowButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(220, 220, 100);</string>
              </property>
              <property name="text">
               <string>yellow</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="cWhiteButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>white</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="cBlackButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(0,0,0);</string>
              </property>
              <property name="text">
               <string>black</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QRadioButton" name="cLightButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(200, 200, 200);</string>
              </property>
              <property name="text">
               <string>light</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QRadioButton" name="cDarkButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(70, 70, 70);</string>
              </property>
              <property name="text">
               <string>dark</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="albumSortingGroupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Sort album:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="albumsSortAButton">
              <property name="text">
               <string>Alphabetical</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="albumsSortDButton">
              <property name="text">
               <string>By date</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Orientation:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="orientationGridLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="orientationLButton">
              <property name="text">
               <string>Landscape</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="orientationPButton">
              <property name="text">
               <string>Portrait</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="orientationAButton">
              <property name="text">
               <string>Auto</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Show track lenght in library view:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="showTrackLenghtYButton">
              <property name="text">
               <string>Yes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="showTrackLenghtNButton">
              <property name="text">
               <string>No</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="langBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Language:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="engLangButton">
              <property name="text">
               <string>English</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ruLangButton">
              <property name="text">
               <string>Russian</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
